cluster/engine.go

```diff
@@ -478,9 +483,20 @@ func (e *Engine) updateClientVersionFromServer(serverVersion string) {
 		e.apiClient.UpdateClientVersion("1.23")
 	case versions.LessThan(serverVersion, "1.13"):
 		e.apiClient.UpdateClientVersion("1.24")
-	default:
+	case versions.LessThan(serverVersion, "1.13.1"):
 		e.apiClient.UpdateClientVersion("1.25")
+	case versions.LessThan(serverVersion, "17.03.1") || serverVersion == "1.13.1":
+		e.apiClient.UpdateClientVersion("1.26")
+	case versions.LessThan(serverVersion, "17.04"):
+		e.apiClient.UpdateClientVersion("1.27")
+	case versions.LessThan(serverVersion, "17.05"):
+		e.apiClient.UpdateClientVersion("1.28")
+	case versions.LessThan(serverVersion, "17.06"):
+		e.apiClient.UpdateClientVersion("1.29")
+	default:
+		e.apiClient.UpdateClientVersion("1.30")
 	}
+	fmt.Println("FINAL CLIENT VERSION", e.apiClient.ClientVersion())
```

Comment:
Thanks for the improvement. And I hope to more details of this line of code. What is the usage of this?